### PR TITLE
Minikube upgrade

### PR DESCRIPTION
**Pull request by smebberson**
This PR contains:

- Updated syntax for a newer version of Minikube.
- Support for a new `start` workflow (i.e. `c workflow start`).

## Related PRs

- idearium/idearium#18

## Setup

- [x] Pull down this PR.
- [x] Execute `yarn link`.
- [x] In a project you want to test it with execute `yarn link @idearium/cli`.

## Testing

- [x] Visit the related PR and go through the steps.
- [x] Execute `c mk stop && c mk start` to make sure the updated `c mk start` syntax works.

### CODE REVIEW

**Comment by allanchau** [comment]
@smebberson Is it possible to detect if `yarn` has been run before `c workflow ...`? I just did that by accident and it errored with `Error: The start workflow doesn't exist. Create it at './devops/workflows/start.js'.`.

File: bin/lib/c-workflow.js [hunk 1]
```diff
@@ -54,21 +62,26 @@ const get = (name = '') => {
...
 
 };
 
 const getAll = () => workflows
     // Try and load the workflow
-    .map(workflow => Object.assign(workflow, { fn: get(workflow.name) }))
+    .map(workflow => Object.assign(workflow, get(workflow.name)));
     // Filter out anything that doesn't exist.
-    .filter(workflow => (workflow.fn !== null));
+    // .filter(({ include }) => (include));
```
Comment (allanchau) [inline bin/lib/c-workflow.js]: Can this line and the comment above be removed now?

**Comment by smebberson** [comment]
> @smebberson Is it possible to detect if `yarn` has been run before `c workflow ...`? I just did that by accident and it errored with `Error: The start workflow doesn't exist. Create it at './devops/workflows/start.js'.`.

@allanchau, not really. That's just how the previous version worked.

**Comment by smebberson** [comment]
@allanchau, all comments have been addressed.

**Pull request merged**